### ISSUE TITLE
Blogs Page - Fixing "Read More" (Green Box link) identifier

### ISSUE THREAD

**Issue by Abhi-Raj** [photobox/pbx-group-security#88]
![image](https://user-images.githubusercontent.com/33575821/33946447-d360dc0e-dfef-11e7-8802-0cd55e6cc683.png)
Please review the red highlighted button and fix it with "Read more" tag or something better for the blogs.

Also, identify if there are any other random green box links in the portal and if so identify the root cause (where they are coming from and get rid of those).

**Comment by fzipi** [comment]
Well, I found the problem. i18n.

@DinisCruz Reviewing the theme, seems that the theme was copied from the universal directory to layout directly.

Usually the way of doing this is a little different: I normally use the original theme unchanged, and then alter layouts and static images from the base directory. I use git submodules for updating the theme, but ymmv. 

In summary: copied from the universal theme the i18n directory and relevant files and that's it. By relevant I mean 'en & fr & es'. Don't know if the content will be translated at some point, but can't do any damage to have the base text in various languages.

I made the fix (d1f2637210807c42e8cfd71864824ea31adeecae) and pushed to master.
